modules/apps/object/object-service/src/main/java/com/liferay/object/internal/validation/rule/FunctionObjectValidationRuleEngineImpl.java

```diff
@@ -156,6 +158,10 @@ private JSONObject _getPayloadJSONObject(Map<String, Object> objectEntry) {
 	private JSONFactory _jsonFactory;
 
 	private String _key;
+
+	@Reference
+	private Language _language;
```

Comment:
@mateussgn here we don't need use `_language` here we can just pass the `_name`.